src/reducers/operationsNew.ts

```diff
@@ -17,13 +17,13 @@ export interface TrustPayload {
   to_mid: Mid;
 }
 
-export interface APIOperationBase {
+export interface ApiOperationBase {
```

Comment:
I'm not really a fan of calling this `ApiOperation` as it implies to me an operation on the API itself (i.e. posting a Trust or something). Could we just call it `Operation`?